gdal/frmts/sentinel2/sentinel2dataset.cpp

```diff
@@ -1386,9 +1473,11 @@ char** SENTINEL2GetL1BGranuleMetadata( CPLXMLNode* psMainMTD )
 static CPLString SENTINEL2GetTilename(const CPLString& osGranulePath,
                                       const CPLString& osGranuleName,
                                       const CPLString& osBandName,
+                                      const CPLString& pszProductURI = CPLString(),
```

Comment:
osProductURI

Reply:
All variables have been renamed according to guidelines